.travis.yml

```diff
@@ -45,6 +45,9 @@ cache:
 before_install:
   - export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-arm64"
   - export PATH="$JAVA_HOME/bin:$PATH"
+  - export MEMORYMB=2048
+  - export DIRECTMEMORYMB=5120
```

Comment:
One note about `DIRECTMEMORYMB`. Do you think `4500M` is really small?
GitHub Actions has the same [7Gb](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) memory limit and these builds pass amost always.